buzsaki_lab_to_nwb/yuta_code_visual_cortex/convert_yuta_vc.py

```diff
@@ -0,0 +1,59 @@
+"""Authors: Heberto Mayorquin and Cody Baker."""
+from pathlib import Path
+
+from nwb_conversion_tools.utils.metadata import load_metadata_from_file
+from nwb_conversion_tools.utils.json_schema import dict_deep_update
+
+from buzsaki_lab_to_nwb import YutaVCNWBConverter
+
+# data_path = Path("/home/heberto/globus_data")
+data_path = Path("/shared/catalystneuro/Buzsaki/SenzaiY")
+nwb_output_path = Path("/home/jovyan/BuzsakiNWB/SenzaiY")
+metadata_path = Path("metadata.yml")
+
+
+stub_test = True
+conversion_factor = 0.195  # Intan
+
+session_list = [
+    session
+    for subject in data_path.iterdir()
+    if subject.is_dir() and "YMV" in subject.name
+    for session in subject.iterdir()
+]
+session_list = [session for session in session_list if session.is_dir()]
+
+subject_genotypes = dict(YMV04="CaMKII-Cre::Ai32", YMV07="CaMKII-Cre::Ai35")
+subject_genotypes.update({f"YMV{subject_num}": "Ai35" for subject_num in ["01", "02", "03"]})
+subject_genotypes.update(
+    {f"YMV{subject_num}": "PV-Cre::Ai32" for subject_num in ["05", "09", "10", "11"] + [str(x) for x in range(13, 20)]}
+)
+subject_genotypes.update({f"YMV{subject_num}": "VGAT-Cre::Ai32" for subject_num in ["06", "08", "12"]})
+
+for session_path in session_list:
+    print(f"Processsing {session_path}...")
+    session_name = session_path.stem
+    subject_name = session_path.parent.name
+    dat_file_path = session_path / f"{session_name}.dat"
+    eeg_file_path = session_path / f"{session_name}.eeg"
+    nwbfile_path = nwb_output_path / f"{session_name}.nwb"
+
+    source_data = dict(
+        NeuroscopeRecording=dict(file_path=str(dat_file_path), gain=conversion_factor),
+        NeuroscopeLFP=dict(file_path=str(eeg_file_path), gain=conversion_factor),
+        YutaVCBehavior=dict(folder_path=str(session_path)),
+        PhySorting=dict(folder_path=str(session_path), exclude_cluster_groups=["noise", "mua"]),
```

Comment:
should `exclude_cluster_groups=["noise", "mua"]` be default for this interface?

Reply:
Yep; paper refers to a certain number of units detected over all sessions, and that number only corresponds to the 'good' units identified by Phy. We've also never written the MUA units indicated by Neuroscope in the past, either.